.github/workflows/prepare_release.yml

```diff
@@ -0,0 +1,27 @@
+name: Prepare Release
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Version to release
+        required: true
+      force:
+        description: Force a release even when there are release-blockers (optional)
+        required: false
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    name: "Release a new version"
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GH_RELEASE_PAT }}
+          fetch-depth: 0
+      - name: Prepare release
+        uses: getsentry/action-prepare-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_RELEASE_PAT }}
+        with:
+          version: ${{ github.event.inputs.version }}
+          force: ${{ github.event.inputs.force }}
+          
```

Comment:
`l`: No newline at end of file
<img width="184" alt="Screen Shot 2021-06-22 at 11 28 52" src="https://user-images.githubusercontent.com/2443292/122900487-06811f00-d34d-11eb-902c-e9c4a9e55103.png">